src/api/index.ts

```diff
@@ -18,6 +32,8 @@ const handleRequestErrors = (req: Request, res: Response): boolean => {
   return false;
 };
 
+const handleNodeFSErrors = (error: Error) => error.name.startsWith('ERR');
```

Comment:
Despite its name, this function does not handle file system errors. It only checks whether error.name starts with `ERR`. According to the Node documentation, it is probably `SystemError` that will be thrown:

https://nodejs.org/api/errors.html#errors_class_systemerror

So a more explicit test might be:

```ts
const isSystemError = (error: unknown) => error instanceof isSystemError;
```
(But not sure if this still warrants a separate function.)